pjmedia/src/pjmedia/stereo_port.c

```diff
@@ -183,20 +183,18 @@ static pj_status_t stereo_get_frame(pjmedia_port *this_port,
     if (status != PJ_SUCCESS)
 	return status;
 
+    // If downstream provides non PCM frames, skip conversion and return silence
+    // For example, empty ConfBridge gives PJMEDIA_FRAME_TYPE_NONE
     if (tmp_frame.type != PJMEDIA_FRAME_TYPE_AUDIO) {
-	frame->type = tmp_frame.type;
-	frame->timestamp = tmp_frame.timestamp;
-	frame->size = tmp_frame.size;
-	if (tmp_frame.size && tmp_frame.buf == sport->get_buf)
-	    pj_memcpy(frame->buf, tmp_frame.buf, tmp_frame.size);
+	pj_bzero(frame->buf, frame->size);
```

Comment:
This will basically zero all non-audio frames. If you only want to handle TYPE_NONE, I would suggest changing it to:
```
    if (tmp_frame.type != PJMEDIA_FRAME_TYPE_AUDIO) {
	frame->type = tmp_frame.type;
	if (tmp_frame.type == PJMEDIA_FRAME_TYPE_NONE) {
	    pj_bzero(frame->buf, frame->size);
	} else {
	    frame->timestamp = tmp_frame.timestamp;
	    frame->size = tmp_frame.size;
	    if (tmp_frame.size && tmp_frame.buf == sport->get_buf)
	        pj_memcpy(frame->buf, tmp_frame.buf, tmp_frame.size);
	}
	return PJ_SUCCESS;
    }
```

Reply:
This was my first guess, but then I wondered who would put StereoPort on non PCM path ?
It can't deal others than TYPE_AUDIO frames.

Reply:
I wonder the same thing, but same as above, when the code has been around for so many years, I tend to try not to change it unless it's broken (i.e. only fix the broken part and leave the unbroken one as is).